vpr/src/place/net_cost_handler.h

```diff
@@ -45,7 +45,7 @@ enum e_cost_methods {
  * @param timing_delta_c
  * @return The number of affected nets.
  */
-int find_affected_nets_and_update_costs(
+void find_affected_nets_and_update_costs(
```

Comment:
Check if this function modifies (and doesn't clear/reset) the ts_* variables; if it changes their state (and leaves it changed) it should be mentioned briefly here.